src/pages/en/editor-setup.md

```diff
@@ -31,7 +31,10 @@ Our amazing community maintains several extensions for other popular editors, in
 - [Nova Extension](https://extensions.panic.com/extensions/sciencefidelity/sciencefidelity.astro/) <span style="margin: 0.25em;"><Badge variant="neutral">Community</Badge></span> - Provides syntax highlighting and code completion for Astro inside of Nova
 - Neovim [LSP](https://github.com/neovim/nvim-lspconfig/blob/master/doc/server_configurations.md#astro) and [TreeSitter](https://github.com/virchau13/tree-sitter-astro) Plugins <span style="margin: 0.25em;"><Badge variant="neutral">Community</Badge></span> - Provides syntax highlighting, treesitter parsing, and code completion for Astro inside of Neovim
 
-Unfortunately, [WebStorm](https://www.jetbrains.com/webstorm/) does not support Astro's language server. Visit the relevant [JetBrains support issue](https://youtrack.jetbrains.com/issue/WEB-52015/Astro-Language-Support) to upvote the ticket, track the progress, and find community workarounds.
+
```

Comment:
I’d love to see a subheading here to make clear that this content is not directly related to the list items.

```suggestion
### JetBrains IDEs

```

Otherwise, LGTM!

Reply:
Yeah, I get that and I like it visually. But, part of me is thinking, why do THEY get their own heading when they can't support us!  🤔

Reply:
We’re calling them out! Publicly shamed!